package-lock.json

```diff
@@ -1,8012 +0,0 @@
-{
```

Comment:
Arguments given for the _.gitignore_ also suggest that this file be kept under version control.